packages/rpc-graphql/src/loaders/account.ts

```diff
@@ -32,16 +32,14 @@ async function loadAccount(
 
 function createAccountBatchLoadFn(rpc: Rpc<GetAccountInfoApi>) {
     const resolveAccountUsingRpc = loadAccount.bind(null, rpc);
-    return async (accountQueryArgs: readonly AccountLoaderArgs[]) => {
-        return await Promise.all(
-            accountQueryArgs.map(async args => await resolveAccountUsingRpc(applyDefaultArgs(args))),
-        );
-    };
+    return async (accountQueryArgs: readonly AccountLoaderArgs[]) =>
+        Promise.all(accountQueryArgs.map(async args => resolveAccountUsingRpc(applyDefaultArgs(args))));
```

Comment:
I thought there were some implications of mismatching `async`/`await` when errors get thrown and stacktraces get produced. Here, you've written an `async` function but returned a `Promise` without awaiting it.

Now I'm curious.

https://stackblitz.com/edit/stackblitz-starters-rkaxay?file=index.js

Hmm… doesn't look like there's a difference.

The existence of these rules makes me think this used to be a problem

* https://eslint.org/docs/latest/rules/require-await
* https://eslint.org/docs/latest/rules/no-return-await

Reply:
It does say "It can now be slower to remove `await` rather than keeping it", but I always thought as long as it was returning a promise it doesn't matter, since it will be awaited at the call site. Seems this wasn't always the case?

Reply:
But seems like V8 handles this, as per your example.